themes/digital.gov/src/js/glossary.js

```diff
@@ -0,0 +1,239 @@
+    }
+    const terms = await response.json();
+    terms.forEach((term, i) => {
+      const opts = {
+        term: term.term,
+        definition: term.definition,
+        definitionClass: classes.definitionClass,
+        glossaryItemClass: classes.glossaryItemClass,
+        termClass: classes.termClass,
+        iconClass: classes.iconClass,
+        termId: i,
+      };
+      this.listElm.insertAdjacentHTML("beforeend", itemTemplate(opts));
+    }, this);
+
+    const glossaryID = selectors.glossaryID.slice(1);
+    const listClass = selectors.listClass.slice(1);
+    const searchClass = selectors.searchClass.slice(1);
+    const options = {
+      valueNames: [classes.termClass],
+      listClass,
+      searchClass,
+    };
+    this.list = new List(glossaryID, options);
+
+    // When last term is reached, focus top of glossary
+    this.addEventListener(
+      this.listElm.lastChild.firstElementChild,
+      "keydown",
+      (e) => {
+        if (e.keyCode === KEYCODE_TAB) {
+          this.search.focus();
+        }
+      }
+    );
+
+    // When first item (close button) is reached, focus top of glossary
+    this.addEventListener(this.closeBtn, "keydown", (e) => {
+      if (e.keyCode === KEYCODE_TAB) {
+        this.search.focus();
+      }
+    });
+  } catch (e) {
+    // No glossary on this page
+  }
+};
+
+// https://davidwalsh.name/element-matches-selector
+function selectorMatches(el, selector) {
+  const p = Element.prototype;
+  const f =
+    p.matches ||
+    p.webkitMatchesSelector ||
+    p.mozMatchesSelector ||
+    p.msMatchesSelector ||
+    function selectorMatch(s) {
+      return [].indexOf.call(document.querySelectorAll(s), this) !== -1;
+    };
+  return f.call(el, selector);
+}
+
+// Toggle glossary term visibility
+Glossary.prototype.handleTermTouch = function handleTermTouch(e) {
+  if (
+    selectorMatches(e.target, `.${classes.termClass}`) ||
+    selectorMatches(e.target, `.${classes.iconClass}`)
+  ) {
+    // Needed to handle icon clicks
+    let { target } = e;
+    if (target.tagName.toLowerCase() === "svg") {
+      target = target.previousElementSibling;
+    } else if (target.tagName.toLowerCase() === "use") {
+      target = target.parentElement.previousElementSibling;
+    }
+
+    const expanded = target.getAttribute("aria-expanded");
+    const termNumber = target.getAttribute("aria-controls");
+    if (expanded === "false") {
+      target.setAttribute("aria-expanded", "true");
+      document
+        .querySelectorAll(`#${termNumber}`)
+        .forEach((el) => el.setAttribute("aria-hidden", "false"));
+    } else {
+      target.setAttribute("aria-expanded", "false");
+      document
+        .querySelectorAll(`#${termNumber}`)
+        .forEach((el) => el.setAttribute("aria-hidden", "true"));
+    }
+  }
+};
+
+// Toggle glossary visibility
+Glossary.prototype.toggle = function toggle() {
+  const method = this.isOpen ? this.hide : this.show;
+  method.apply(this);
+};
```

Comment:
We could use a more modern method like [Element: toggleAttribute() method | MDN](https://developer.mozilla.org/en-US/docs/Web/API/Element/toggleAttribute) or [HTMLDetailsElement | MDN](https://developer.mozilla.org/en-US/docs/Web/API/HTMLDetailsElement/toggle_event).